backend/collaboration-service/user_verification.py

```diff
@@ -6,7 +6,8 @@
     raise ValueError('USER_SERVICE_URL environment variable not set')
 
 def authenticate(authorization_header) -> str | None:
-    response = requests.get(f"{USER_SERVICE_URL}/auth/verify-token", headers={'Authorization': authorization_header})
+    response = requests.get(f"{USER_SERVICE_URL}/auth/verify-token", headers={'authorization': authorization_header})
+    print(response)
```

Comment:
Please remove printing statements, and use `logging` statements instead.

Reply:
Yeah i will push the final commit, still working on cursor update